Integrations/PaloAltoNetworks_Traps/PaloAltoNetworks_Traps_test.py

```diff
@@ -0,0 +1,121 @@
+        'trapsVersion': '6.1.0.13046',
+        'contentVersion': '63-10484', 'ip': 'xxx.xx.xx.xxx',
+        'computerSid': 'S-1-5-21-202186053-2642234773-3690463397', 'installStatus': 'installed',
+        'installTime': '2019-09-05T10:51:35.000Z',
+        'distributionId': {'guid': 'afbf42010b6233624ffc20ca95d51ff3'},
+        'compromised': False, 'alias': None,
+        'osVersion': '10.0.14393',
+        'osProductType': 'server',
+        'osProductName': '',
+        'is64': True,
+        'lastSeen': '2019-09-24T15:10:21.000Z',
+        'provisioning': {'name': None, 'domain': None, 'ip': None},
+        'lastUser': 'Administrator',
+        'isLicensed': True,
+        'vdi': 'none',
+        'isolationStatus': 'isolated',
+        'wsConnected': False,
+        'capabilities': {
+            'quarantine': True,
+            'networkIsolation': True,
+            'terminateProcess': True,
+            'fileRetrieval': True,
+            'liveTerminal': True,
+            'scriptExecution': False
+        }
+    }
+    endpoint_data = parse_data_from_response(resp_obj, 'get_endpoint_by_id')
+    expected_endpoint_data = {
+        'ID': 'd3339851f18f470182bf2bf98ad5db4b',
+        'Name': 'EC2AMAZ-8IEUJEN',
+        'Domain': 'WORKGROUP',
+        'Platform': 'windows',
+        'Status': 'active',
+        'IP': 'xxx.xx.xx.xxx',
+        'ComputerSid': 'S-1-5-21-202186053-2642234773-3690463397',
+        'IsCompromised': False,
+        'OsVersion': '10.0.14393',
+        'OsProductType': 'server',
+        'OsProductName': '',
+        'Is64': True,
+        'LastSeen': '2019-09-24T15:10:21.000Z',
+        'LastUser': 'Administrator'
+    }
+
+    assert endpoint_data == expected_endpoint_data
+
+
+def test_event_quarantine(requests_mock, mocker):
+    mocker.patch.object(demisto, 'params', return_value=integration_params)
+    from PaloAltoNetworks_Traps import event_quarantine
+    event_id = '7dc177a4df1c41b19ca1e67e8573b6be'
+    quarantine_path = f'events/{event_id}/quarantine'
+    mock_resp_json = {'operationId': {'samMessageIds': ['80cf8859df7811e9acbf0245d8e950da']}}
+    requests_mock.post(SERVER_MOCK_URL + quarantine_path, json=mock_resp_json)
+    operations = event_quarantine(event_id)
+    expected_operations = [{
+        'EventID': '7dc177a4df1c41b19ca1e67e8573b6be',
+        'Type': 'event-quarantine',
+        'OperationID': '80cf8859df7811e9acbf0245d8e950da'
+    }]
+    assert expected_operations == operations
+
+
+def test_endpoint_isolate(requests_mock, mocker):
+    mocker.patch.object(demisto, 'params', return_value=integration_params)
+    from PaloAltoNetworks_Traps import endpoint_isolate
+    endpoint_id = 'd3339851f18f470182bf2bf98ad5db4b'
+    isolate_path = f'agents/{endpoint_id}/isolate'
+    mock_resp_json = {'operationId': '458e2003dfb411e9acbf0245d8e950da'}
+    requests_mock.post(SERVER_MOCK_URL + isolate_path, json=mock_resp_json)
+    operation_obj = endpoint_isolate(endpoint_id)
```

Comment:
I think it is better to unit test the `_command` functions, they have more coverage

Reply:
This is true but it seems like that is not really "unit testing" no? Plus it is widely tested in the test playbook and to OOB playbooks.
If you still think we should change the unit tests and replace the functions test to commands we can do that together with the package changes we said we can do after the release.